presto-main/src/test/java/io/prestosql/operator/TestOperatorAssertion.java

```diff
@@ -52,7 +52,7 @@ public void testToPagesWithBlockedOperator()
     {
         Operator operator = new BlockedOperator(Duration.valueOf("15 ms"));
         List<Page> pages = OperatorAssertion.toPages(operator, emptyIterator());
-        Assert.assertEquals(pages, ImmutableList.of());
+        assertEquals(pages, ImmutableList.of());
```

Comment:
nit: does not belong to `Access static methods on declared class` 

(same in TestPinotClient)